core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/NamespaceChangeStreamListener.java

```diff
@@ -254,7 +254,7 @@ void storeNextEvent() {
           return;
         }
 
-        logger.info(String.format(Locale.US,
+        logger.debug(String.format(Locale.US,
             "NamespaceChangeStreamListener::stream ns=%s event found: op=%s id=%s",
```

Comment:
Seems like a strange drive-by so just making sure you meant to make this change

Reply:
Yeah I did this because the log was extremely noisy when getting thousands of events.